src/snapshots/new_nu_parser__test__node_output@for_break_continue.nu.snap

```diff
@@ -47,34 +48,35 @@ input_file: tests/for_break_continue.nu
 0: int
 1: int
 2: ()
-3: unknown
-4: unknown
-5: unknown
-6: unknown
-7: unknown
-8: unknown
-9: unknown
-10: unknown
-11: unknown
+3: int
+4: int
+5: int
+6: int
+7: list<int>
+8: int
+9: forbidden
+10: int
+11: bool
 12: unknown
 13: unknown
-14: unknown
-15: unknown
-16: unknown
-17: unknown
-18: unknown
+14: oneof<(), unknown>
```

Comment:
The type of missing `if` branch is `nothing`. So, `if $spam { 1 }` should have a return type of `oneof<int, nothing>`.

Reply:
That `()` is from the else branch. The if branch does not have a type because continue and break are not checked/parsed properly.

Reply:
Yeah, I meant the else branch. It should be `nothing`, not `()` coming from the missing else branch. Try `if false { 1 } | describe` in Nushell, it'll print `nothing`, so we should match the behavior.